### PR TITLE
Add dlocal, ebanx, fortumo methods

### PR DESCRIPTION
**Pull request by steve-gr4vy**
# Description

Add methods for new connectors

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn test` to make sure my changes pass all linters
- [x] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

**Review by ipeluffo — COMMENTED** [review]
it seems that some of these are missing in https://github.com/gr4vy/core-api/blob/6e5f4145e72c15c36f7e947af2375bbc0c149a83/src/api/types.py#L11-L35

Maybe we should add them in Core API before this PR is merged.

**Comment by steve-gr4vy** [comment]
> it seems that some of these are missing in https://github.com/gr4vy/core-api/blob/6e5f4145e72c15c36f7e947af2375bbc0c149a83/src/api/types.py#L11-L35
> 
> Maybe we should add them in Core API before this PR is merged.

@ipeluffo correct, the missing will be part of a dlocal PR: e.g. https://github.com/gr4vy/core-api/blob/add-dlocal-dlocal/src/api/types.py#L36-L40

File: reference/enums/methods.yaml [hunk 1]
```diff
@@ -12,3 +12,16 @@ enum:
...
+  - gcash
+  - grabpay
+  - ovo
+  - paymaya
+  - rabbitlinepay
+  - shopeepay
+  - truemoney
+  - klarna
+  - trustly
+  - pix
```
Comment (ipeluffo) [inline reference/enums/methods.yaml:27]: [minor] Should we sort the list alphabetically?

**Review by ipeluffo — DISMISSED** [review]
Approving since my comment is a minor one but I'd expect this to be merged after https://github.com/gr4vy/core-api/blob/add-dlocal-dlocal/src/api/types.py#L36-L40 has been merged

File: reference/enums/methods.yaml (hunk 1 above)
Comment (cbetta) [reply reference/enums/methods.yaml]: @steve-gr4vy gone ahead and sorted it.

**Review by cbetta — APPROVED** [review]

**Pull request merged**